Doco - describe refresh_token process (#1779)
praw-dev/praw: docs/tutorials/refresh_token.rst

.. literalinclude:: ../examples/obtain_refresh_token.py
    :language: python

This script assumes you have configured your application's ``redirect uri`` to
``localhost:8080``

When you execute this script interactively:

> - You will be prompted for a comma-separated list of scopes

Reviewer: ```suggestion
- You will be prompted to provide a comma-separated list of scopes
```

> - You will be given a URL that will take you through the auth follow

Reviewer: ```suggestion
- You will be given a URL that will take you through the auth flow on Reddit
```

> - Reddit will ask you for user authentication and ask you to grant the application
>   permissions

Reviewer: ```suggestion
- Open the provided link in your browser and Reddit will ask you for permission to grant
  the application permissions to the scopes requested
```

> - On completion, the user will have a new authorized application configured

Reviewer: ```suggestion
- After clicking allow, you will have a new authorized application configured
```

> - A refresh token is displayed in the browser and on the command line

Reviewer: ```suggestion
- You will be redirected to another page where your refresh token will be displayed and
  will be printed to the command line
```

> 

Reviewer: What do you think about making this a numbered list?
